main.go

```diff
@@ -85,24 +88,34 @@ func main() {
 		os.Exit(1)
 	}
 
-	cfg := config.Default()
+	var cfg unversioned.EraserConfig
+	v1alpha1Config := v1alpha1Config.Default()
 	if configFile != "" {
-		o, err := options.AndFrom(ctrl.ConfigFile().AtPath(configFile).OfKind(cfg))
+		type apiVersion struct {
+			APIVerison string `json:"apiVersion"`
+		}
+		o, err := options.AndFrom(ctrl.ConfigFile().AtPath(configFile).OfKind(v1alpha1Config))
 		if err != nil {
 			setupLog.Error(err, "configuration is either missing or invalid")
 			os.Exit(1)
 		}
 
 		options = o
+		if err := v1alpha1.Convert_v1alpha1_EraserConfig_To_unversioned_EraserConfig(v1alpha1Config, &cfg, nil); err != nil {
+			setupLog.Error(err, "CONVERSION FAILED")
+			os.Exit(1)
+		}
```

Comment:
@sozercan This seems like a hack around what's supposed to happen. On the plus side we get the auto-generated conversion functions, but I would assume that there's a better way to do this.

At the very worst we can remove the `ControllerManagerConfigurationSpec` as a member of EraserConfig and instead require an `apiVersion` that we manage ourselves. It would have to be added to the helm chart.